config/routes.rb

```diff
@@ -14,6 +14,17 @@
   # Defines the root path route ("/")
   root "home#index"
 
+  # Daily Flow routes
+  get "daily_flow/start", to: "daily_flow#start", as: :daily_flow_start
+  get "daily_flow/check_in", to: "daily_flow#check_in", as: :daily_flow_check_in
+  post "daily_flow/check_in", to: "daily_flow#save_check_in", as: :daily_flow_save_check_in
+  get "daily_flow/affirmation", to: "daily_flow#affirmation", as: :daily_flow_affirmation
+  get "daily_flow/gratitude", to: "daily_flow#gratitude", as: :daily_flow_gratitude
+  post "daily_flow/gratitude", to: "daily_flow#save_gratitude", as: :daily_flow_save_gratitude
+  get "daily_flow/reflection", to: "daily_flow#reflection", as: :daily_flow_reflection
+  post "daily_flow/reflection", to: "daily_flow#save_reflection", as: :daily_flow_save_reflection
+  get "daily_flow/completion", to: "daily_flow#completion", as: :daily_flow_completion
+
```

Comment:
There should be something at 'get "daily_flow'' since clicking on that breadcrumb give a routing error.